Editor/Completion/XmlCompletionCommitManager.cs

```diff
@@ -38,10 +38,39 @@ public CommitResult TryCommit (IAsyncCompletionSession session, ITextBuffer buff
 			}
 
 			switch (kind) {
+			case XmlCompletionItemKind.SelfClosingElement: {
+					if (typedChar != '>') {
+						string insertionText = $"{item.InsertText}/>";
+						Insert (session, buffer, insertionText);
+						ShiftCaret (session, 2, XmlCaretDirection.Left);
+						return CommitResult.Handled;
+					} else {
+						return CommitResult.Unhandled;
+					}
+				}
+			case XmlCompletionItemKind.Element: {
+					string insertionText = $"{item.InsertText}></{item.InsertText}>";
```

Comment:
If the completion command was invoked with ctrl-space, the `InsertText` will contain the `<`. You should trim any leading `<` from the value used in the closing tag.

Reply:
I tried checking when the completion was invoked with ctrl-space, it doesn't introduce any unnecessary `<` in the closing tag or opening tag

Reply:
Resolved using `.Trim`